types/node-downloader-helper/index.d.ts

```diff
@@ -51,4 +93,10 @@ export class DownloaderHelper extends EventEmitter {
     pause(): Promise<boolean>;
     resume(): Promise<boolean>;
     stop(): Promise<boolean>;
+    pipe(readableStream: StreamReadable, options?: any): StreamReadable; // Add pipes to the pipe list that will be applied later when the download starts
+    unpipe(readableStream?: StreamReadable): void; // Unpipe an stream , if a stream is not specified, then all pipes are detached.
+    getDownloadPath(): string;	// gets the full path where the file will be downloaded (available after the start phase)
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/node-downloader-helper/index.d.ts;
  Line: 98;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.